src/components/inputs/async-creatable-select-input/async-creatable-select-input.js

```diff
@@ -147,5 +179,9 @@ export class AsyncCreatableSelectInput extends React.Component {
 }
 
 const Wrapped = injectIntl(AsyncCreatableSelectInput);
-Wrapped.isTouched = AsyncCreatableSelectInput.isTouched;
+addStaticFields(Wrapped, {
+  ...defaultComponents,
+  ...customizedComponents,
+  isTouched: AsyncCreatableSelectInput.isTouched,
+});
```

Comment:
I wonder how this would influence the tree-shaking capabilities. But these components are going to be bundled with `react-select` anyways, so it probably doesn't matter at all.